src/widgets/Header/ui/Header/Header.tsx

```diff
@@ -12,18 +13,30 @@ type THeaderProps = {
 };
 
 const Header: FC<THeaderProps> = props => {
+  const location = useLocation();
+
+  const isErrorPage = (pathname: string): boolean => {
+    return pathname !== '/' && pathname !== '/product';
+  };
```

Comment:
This function does not check whether this is an error page or not.
Also u created the same function in Logo.tsx